frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -93,6 +115,29 @@ export class ResourceMenuBase extends React.Component<Props, State> {
             resource.path.toLowerCase().indexOf(search.toLowerCase()) > -1
         );
 
+        const sort = this.state.sortAsc ? "fa fa-caret-up" : "fa fa-caret-down";
+
+        const sortResource = this.state.sortActive === 'resource' ? sort : ""
+
+        const sortProgress = this.state.sortActive === 'progress' ? sort : "";
+
+        function getProgress(el){
+            let approvedStrings = el.approvedStrings,
+                stringsWithWarnings = el.stringsWithWarnings,
+                totalStrings = el.totalStrings;
```

Comment:
We use `const` instead of `let` where possible, put `const` in front of each variable and end the line with a colon. Here however I'd remove these lines and use `el.X` directly.